src/containers/MsgInfo/MsgInfo.js

```diff
@@ -44,36 +41,21 @@ class MsgInfo extends Component {
     return props.info;
   }
 
-  /**
-   * @description - This method clear msg info
-   * @returns {object} null
-   * @memberof MsgInfo
-   */
-  close() {
-    this.props.clearMsgInfo();
-  }
-
   /**
    *
    * @description - This method renders the jsx for this component
    * @returns {jsx} - jsx
    * @memberof MsgInfo
    */
   render() {
-    const { success, info } = this.state;
+    const { success, message } = this.state;
 
-    const messages = info.map((data, index) => (<li key={index} className="info">{data}</li>));
+    const messages = message.map((data, index) => (<li key={index} className="info">{data}</li>));
 
     return (
-      <div className={info.length ? '_show' : '_hide'}>
+      <div className={message.length ? '_show' : '_hide'}>
```

Comment:
react-materilize implements in its own `show` and `hide` css property, so i added to underscore to both `show` and `hide` to avoid conflict